app/controllers/hyrax/admin/permission_template_accesses_controller.rb

```diff
@@ -42,18 +27,6 @@ def after_destroy_success
         end
       end
 
-      def after_destroy_error
-        if source.admin_set?
```

Comment:
I believe this is still needed.  There can still be an error that prevents the template access from being deleted.  If this happens, it should still redirect to the correct edit form and flash the error message.

Reply:
I've fixed it.